.github/workflows/pytest.yml

```diff
@@ -99,8 +97,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8", "3.9", "3.10"]
-        # python-version: ["3.8"]
+        # python-version: ["3.8", "3.9", "3.10"]
+        python-version: ["3.8"]
```

Comment:
no testing on newer python versions anymore?

Reply:
oh thanks, nearly forgot to enable that again